src/example_package/gui/app.py

```diff
@@ -58,8 +64,38 @@ class LoadCSV(BoxLayout):
     pass
 
 
-class ShowParticipants(BoxLayout):
-    pass
+# name,surname,age_category,age,weight_category,weight,country,image_url
+
+class ShowParticipants(ScrollView):
+
+    def __init__(self, **kwargs):
+        super(ShowParticipants, self).__init__(**kwargs)
+        self.participants_data = pd.read_csv('sample.csv')
+        self.generate_layout()
+
+    def generate_layout(self):
+
+        layout = GridLayout(cols=7, spacing=26, size_hint_y=None, padding=[dp(20), dp(20)])
+        layout.bind(minimum_height=layout.setter('height'))
+
+        headers = ['Name', 'Surname', 'Age Category', 'Age', 'Weight Category', 'Weight', 'Country']
```

Comment:
Zastanawiam się, czy nie lepiej byłoby odnieść się już do zdefiniowanej tablicy z wymaganymi kolumnami, żeby nie powielać kodu. Na przykład odnieść się do config.py do "required_columns", tylko żeby ładnie printowało to zrobić tak: `headers = [header.replace('_', ' ').title() for header in constraints["required_columns"][:-1]]` no i zaimportować `constraints` z csv_reader.config